.github/workflows/kuksa_databroker_build.yml

```diff
@@ -127,7 +127,7 @@ jobs:
       working-directory: ${{github.workspace}}/
       run: |
         cargo install cross cargo-license cargo-cyclonedx
-        pip install "git+https://github.com/eclipse-kuksa/kuksa-common.git@6f3d7627760582d8ba83cc8a0f7449d00fffee84#subdirectory=sbom-tools"
+        pip install "git+https://github.com/eclipse-kuksa/kuksa-common.git@de5f39846cfd5a0a3bce7046f598645f006bdb9d#subdirectory=sbom-tools"
```

Comment:
I pushed some tags (`1` and `1.0`) to kuksa-common so the line below should work as well. The advantage with that is that if we do a similar smaller update in kuksa-common (and update `1` and pushes `1.1`) we would not need to change reference here. I described the approach here: https://github.com/eclipse-kuksa/kuksa-common/wiki/Release-Process

But with that said, specifying specific commit should work as well

`pip install "git+https://github.com/eclipse-kuksa/kuksa-common.git@1#subdirectory=sbom-tools"`